### PR TITLE
Update documentation to get sample data directly from naip source

### PR DESCRIPTION
**Pull request by sushilkm**

### CODE REVIEW

File: docs/cataloging-sample-data.md [hunk 1]
```diff
@@ -6,23 +6,36 @@ The steps below walks you through the creation of STAC Collection and STAC Item.
...
-    azcopy copy https://aoigeospatial.blob.core.windows.net/stac/collection_naip_test.json ~/data/collection_naip_test.json
+    wget https://raw.githubusercontent.com/Azure/Azure-Orbital-STAC/main/deploy/sample-data/collection_naip_test.json -P ~/data/collection_naip_test.json
     ```
 
     ```bash
     azcopy copy ~/data/collection_naip_test.json "https://<storage account>.blob.core.windows.net/staccollection/collection_naip_test.json?<SAS-token>" --recursive=true
     ```
 
-2. Once the STAC collection is created, the next steps is to create the [STAC Items](https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md). Addition of the STAC Item to a STAC Collection is two step process.
+2. Generate the SAS token for getting the data from Planetary Computers using the link https://planetarycomputer.microsoft.com/api/sas/v1/token/naip. Clicking on the link will give an output like this:
```
Comment (senthilkungumaraj) [inline docs/cataloging-sample-data.md:20]: I tried accessing this link and was able to get the token as you suggested. But I wanted to understand if there a handshake happening that we aren't seeing that the browser handles. 
Trying to understand if there is a need for an active Azure tenant to use this link.

Comment (sushilkm) [reply docs/cataloging-sample-data.md:20]: I would doubt the requirement of the tenant, the web links are considered public links with a valid sas-token.

File: docs/cataloging-sample-data.md [hunk 2]
```diff
@@ -6,23 +6,36 @@ The steps below walks you through the creation of STAC Collection and STAC Item.
...
+
+```
+{
+    "msft:expiry":"2021-04-08T18:49:29Z",
```
Comment (mandarinamdar) [inline docs/cataloging-sample-data.md:24]: we should obfuscate this token though its invalid....

**Review by mandarinamdar — CHANGES_REQUESTED** [review]

File: docs/cataloging-sample-data.md [hunk 3]
```diff
@@ -6,23 +6,36 @@ The steps below walks you through the creation of STAC Collection and STAC Item.
...
+    "token":"se=2021-04-08T18%3A49%3A29Z&sp=rl&sv=2020-02-10&sr=c&skoid=cccccccc-dddd-4444-aaaa-eeeeeeeeeeee&sktid=***&skt=2021-04-08T17%3A47%3A29Z&ske=2021-04-09T17%3A49%3A29Z&sks=b&skv=2020-02-10&sig=bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb%3D"
+}
+```
+The `token` field is the SAS token. The `msft:expiry` field specifies the time (in UTC) this token expires.
```
[the remark this answers is not in the document]
Comment (mandarinamdar) [reply docs/cataloging-sample-data.md:28]: I would rather obfuscate this token...

[the remark this answers is not in the document]
Comment (senthilkungumaraj) [reply docs/cataloging-sample-data.md:28]: Yes, users should not use this token as-is. But also, the token from the link should be regenerated every 45 minutes to an hour depending on its expiration. There shouldn't be any need for users having to use to more than 45 minutes for our sample data but just calling it out.

**Review by senthilkungumaraj — APPROVED** [review]

File: docs/cataloging-sample-data.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sushilkm) [reply docs/cataloging-sample-data.md]: Done

File: docs/cataloging-sample-data.md (hunk 2 above)
Comment (sushilkm) [reply docs/cataloging-sample-data.md:24]: done.

**Review by mandarinamdar — APPROVED** [review]
LGTM

**Pull request merged**